pilot/helpers/agents/Developer.py

```diff
@@ -223,10 +223,10 @@ def step_human_intervention(self, convo, step: dict):
                 self.get_run_command(convo)
 
             if self.run_command:
-                if (self.project.ipc_client_instance is None or self.project.ipc_client_instance.client is None):
-                    human_intervention_description += color_yellow_bold('\n\nIf you want to run the app, just type "r" and press ENTER and that will run `' + self.run_command + '`')
-                else:
+                if self.project.check_ipc():
```

Comment:
logic is same as before just instead of checking
`if not self.project.check_ipc():`
I swapped it around so that we are checking 
`if self.project.check_ipc():`